Source/Initialization/WarpXInitData.cpp

```diff
@@ -105,6 +105,145 @@ WarpX::PostProcessBaseGrids (BoxArray& ba0) const
     }
 }
 
+void
+WarpX::PrintMainPICparameters ()
+{
+    amrex::Print()<<"-------------------------------------------------------------------------------"<<"\n"; //oshapoval
```

Comment:
No need to add the `//oshapoval` comments - since we use `git` we track when which line comes in and gets modified :)

Reply:
Thanks for spotting! Did not notice while was doing clean up. I will have another round of clean up!